File: CSSamples/Residences/House.cs
using System;

namespace csBookSamples.Residences
{
    class House : Residence
    {
        public void sampleMethod()
        {
            Console.WriteLine("Public method in House class");
            this.protectedMethod();
        }

        public override void message()
        {
            Console.WriteLine("Message from House class");
        }
    }
}

File: JavaSamples/src/com/runninmancoding/bookSamples/classes/residences/House.java
package com.runninmancoding.bookSamples.classes.residences;

public class House extends Residence{
	
	public void sampleMethod(){
		System.out.println("Public method in House class");
		this.protectedResidenceMethod();
	}

	@Override
	public void randomMessage(){
		System.out.println("Random message from House class");
	}
}
